docs/tasks/Docker.md

```diff
@@ -224,7 +229,7 @@ $this->taskDockerRun('wordpress')
 * `interactive()` 
 * `exec($run)` 
 * `volume($from, $to = null)` 
-* `env($variable, $value = null)` 
+* `injectEnv($variable, $value = null)` 
```

Comment:
Maybe the `env` method should take $variable, $value parameters as the existing taskDockerRun does, rather than using array syntax.